internal/bits/bit_array.go

```diff
@@ -433,6 +433,7 @@ func (bA *BitArray) ToProto() *cmtprotobits.BitArray {
 // FromProto sets a protobuf BitArray to the given pointer.
 func (bA *BitArray) FromProto(protoBitArray *cmtprotobits.BitArray) {
 	if protoBitArray == nil {
+		//nolint:wastedassign
```

Comment:
this one is interesting. linter says `bA = nil` does nothing, which I don't think is true